src/components/Sum.jsx

```diff
@@ -1 +1,8 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>;
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
First; you should move the <p> to the new line.

Second; I don't think you need a defaultProp here because you can give default values on the upper lines.
When you are adding { a, b } params, you can give them default values. And by doing that, you should also move them to the following lines.

Reply:
You are right, Thank you!😄

Reply:
Happy to help :)